pyInterface/compareAmplitudeFiles.py

```diff
@@ -43,7 +43,7 @@ def saveTree(path, name, title, lst):
 	for element in lst:
 		arr[0] = element
 		tree.Fill()
-	hist = ROOT.TH1D(name + "_hist", title, 301, -1e-11, 1e-11)
+# 	hist = ROOT.TH1D(name + "_hist", title, 301, -1e-11, 1e-11)
```

Comment:
Why are you making this a comment? The draw command in the next line should be drawing into this histogram, should it not?